### ISSUE TITLE
Fail to resolve swiper@6 when i upgrade 0.1.9

### ISSUE THREAD

**Issue by shaodahong** [web-infra-dev/rspack#2972]
### System Info

codesanbox

### Details

Loading Swiper@6 in version 1.0.8 is normal, but failed after upgrading to version 1.0.9.

```js
 {
      message: 'Failed to resolve ../../core in /project/home/shaodahong/workspace/node_modules/.pnpm/swiper@6.8.4/node_modules/swiper/esm/react/loop.js',
      formatted: 'error[internal]: Resolve error\n' +
        '  ┌─ node_modules/.pnpm/swiper@6.8.4/node_modules/swiper/esm/react/loop.js:3:1\n' +
        '  │\n' +
        "3 │ import Swiper from '../../core';\n" +
        '  │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Failed to resolve ../../core in /project/home/shaodahong/workspace/node_modules/.pnpm/swiper@6.8.4/node_modules/swiper/esm/react/loop.js\n' +
        '\n',
      compilerPath: undefined
    },
```

### Reproduce link

https://codesandbox.io/p/sandbox/rspack-css-loader-forked-6899j1?file=%2Fpackage.json&selection=%5B%7B%22endColumn%22%3A11%2C%22endLineNumber%22%3A17%2C%22startColumn%22%3A11%2C%22startLineNumber%22%3A17%7D%5D

### Reproduce Steps

1. pnpm i
2. node build.js

**Comment by hardfist** [comment]
@shaodahong do you know how to export project in your codesandbox, it's not easy for us to debug in codesandbox

**Comment by shaodahong** [comment]
@hardfist Maybe it can be opened from VS Code. <img width="385" alt="image" src="https://user-images.githubusercontent.com/14831261/235102669-7f253935-5057-44c1-888b-4f41ba8f1c74.png">

**Comment by hardfist** [comment]
> https://codesandbox.io/p/sandbox/rspack-css-loader-forked-6899j1?file=%2Fpackage.json&selection=%5B%7B%22endColumn%22%3A11%2C%22endLineNumber%22%3A17%2C%22startColumn%22%3A11%2C%22startLineNumber%22%3A17%7D%5D

yes, but it's opened in codesandbox machine which makes it slow, do you know better way?

**Comment by shaodahong** [comment]
> > https://codesandbox.io/p/sandbox/rspack-css-loader-forked-6899j1?file=%2Fpackage.json&selection=%5B%7B%22endColumn%22%3A11%2C%22endLineNumber%22%3A17%2C%22startColumn%22%3A11%2C%22startLineNumber%22%3A17%7D%5D
> 
> yes, but it's opened in codesandbox machine which makes it slow, do you know better way?

@hardfist I uploaded the changes to GitHub. Here's the link: https://github.com/shaodahong/rspack-swiper6-test-1.

**Comment by bvanjoi** [comment]
I tested with the main branch and it didn't reproduce, maybe we can wait for version 0.0.10

**Comment by shaodahong** [comment]
Confirm fixed in 0.0.10 version
